RESOURCES.md

```diff
@@ -48,73 +56,81 @@ Below are links to various places you can find the Kentico developer community.
 
 - [Kentico Certified Developers Group on LinkedIn](https://www.linkedin.com/groups/3172023/) - Kentico LinkedIn group for all Kentico certified developers
 
+- [Kentico Xperience Product Suggestions](https://portal.productboard.com/kenticoxperience/) - Product Board site to discuss, recommend, and vote on ideas for the Kentico Xperience platform
+
 ### Kentico Company & Employees on Social Media
 
 #### Kentico on Twitter
 
-- [Kentico](https://twitter.com/Kentico) - Kentico
+- [Kentico](https://twitter.com/Kentico) - Kentico Company account
 
-- [Kentico Kontent](https://twitter.com/KenticoKontent) - Kentico Kontent
+- [Kentico Kontent](https://twitter.com/KenticoKontent) - Kentico Kontent product
 
-- [Kentico Design](https://twitter.com/KenticoDesign) - Kentico's internal designers
+- [Kentico Xperience](https://twitter.com/KenticoXP) - Kentico Xperience product
 
-- [Michal Kadák (Kentico Xperience)](https://twitter.com/MichalKadak) - Product Director - Kentico Xperience
```

Comment:
I removed Michal's stuff throughout the doc. Most of it is product updates for EMS v12 and 13 that won't be as relevant once the product launches.